templates/partials/topnav.html

```diff
@@ -18,5 +21,141 @@
             </div>
         </li>
         {%- endfor -%}
+        <li class="imenu__list__item imenu__list__item--more">
+            <div class="imenu__list__anchor-container">
+                <button type="button" class="imenu__list__anchor" aria-expanded="false">
+                    Mer<span class="imenu__list__anchor-span"></span>
+                    <svg class="imenu__list__anchor-icon-right icon" focusable="false" aria-hidden>
+                        <use xlink:href="#docs-icon-arrow-down"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="ipopupmenu ipopupmenu--vertical">
+                <ul class="ipopupmenu__list ipopupmenu__list--more">
+                    <!-- prettier-ignore -->
+                    {%- for nav in topnav.children -%}
+                    {% set navPath = nav.path | replace("html", "") %}
+                    {% set navPath = navPath | replace(".", "") %}
+                    {% set activePage = navPath in doc.fileInfo.fullPath %}
+                    <li
+                        class="ipopupmenu__list__item {% if activePage %}ipopupmenu__list__item--highlight{% endif %}"
+                        data-item="nav-item-{{loop.index}}"
+                    >
+                        <a href="{{ nav.path | relative(doc) }}"> {{ nav.title }} </a>
+                    </li>
+                    {%- endfor -%}
+                </ul>
+            </div>
+        </li>
     </ul>
 </nav>
+
+<script>
+    function getElementsRefs() {
+        const nav = document.querySelector("#topnav");
+        const moreItem = nav.querySelector(".imenu__list__item--more");
+        const moreButton = moreItem.querySelector("button");
+        const popoverContainer = moreItem.querySelector(".ipopupmenu");
+        const popover = popoverContainer.querySelector(".ipopupmenu__list");
+        const menuItems = Array.from(nav.querySelectorAll(".imenu__list__item[data-item]"));
+        const popoverItems = Array.from(popover.querySelectorAll(".ipopupmenu__list__item[data-item]"));
+        const highlightIndex = menuItems.findIndex((it) => it.classList.contains("imenu__list__item--highlight"));
+        const itemPairs = menuItems.map((menu, index) => ({ menu, popover: popoverItems[index] }));
+
+        return { nav, moreItem, moreButton, popoverContainer, popover, itemPairs, highlightIndex };
+    }
+
+    function setPopoverVisibility(visible, refs) {
+        refs.popover.style.visibility = visible ? "" : "hidden";
+        refs.moreButton.ariaExpanded = visible.toString();
+    }
+
+    function togglePopover(event, refs) {
+        setPopoverVisibility(refs.popover.style.visibility === "hidden", refs);
+        event.stopPropagation();
```

Comment:
Dont know if intentional but if `setPopoverVisibility` fails `event.stopPropagation()` would not be called. Maybe move it so it runs first? Same issue in other places.

Reply:
- [ ] stop propagation earlier at all places

Reply:
Fixed in 0f3e1e2d4757ab8ea9683d076140a672bc403eaf.